clients/cli/src/main/java/org/apache/gravitino/cli/commands/ListSchemaProperties.java

```diff
@@ -54,10 +54,16 @@ public ListSchemaProperties(
   /** List the properties of a schema. */
   @Override
   public void handle() {
-    Schema gSchema = null;
-    try {
-      GravitinoClient client = buildClient(metalake);
-      gSchema = client.loadCatalog(catalog).asSchemas().loadSchema(schema);
+    try (GravitinoClient client = buildClient(metalake)) { // Ensure resource cleanup
+      Schema gSchema = client.loadCatalog(catalog).asSchemas().loadSchema(schema);
+
+      if (gSchema == null) {
+        exitWithError("Schema not found: " + schema);
+        return;
+      }
+
+      Map<String, String> properties = gSchema.properties();
+      printProperties(properties);
```

Comment:
please move the line back to wher ethey were adn do the null check outside of the try block